docs/guides/live-code.mdx

```diff
@@ -85,4 +85,86 @@ export default props => (
 )
 ```
 
+## Using the MDXProvider context for rendering
+
+With the react-live editor you can ensure that your MDX components
+in context are rendered in the preview.  In order to do this you
+can use the MDX custom pragma.
+
+To achieve this, import the pragma:
+
+```js
+import {mdx} from '@mdx-js/react'
+```
+
+Then apply a code transformation for the pragma and add it
+to scope:
+
+```js
+<LiveProvider
+  code={children.trim()}
+  transformCode={code => '/** @jsx mdx */' + code}
+  scope={{ mdx }}
+>
+```
+
+### All together
+
+```js
+/* eslint react/jsx-key: 0 */
+
+import React from 'react'
+import Highlight, {defaultProps} from 'prism-react-renderer'
+import {LiveProvider, LiveEditor, LiveError, LivePreview} from 'react-live'
+import {mdx} from '@mdx-js/react'
+
+export default ({children, className, live, render}) => {
+  const language = className.replace(/language-/, '')
+
+  if (live) {
+    return (
+      <div style={{marginTop: '40px', backgroundColor: 'black'}}>
+        <LiveProvider
+          code={children.trim()}
+          transformCode={code => '/** @jsx mdx */' + code}
+          scope={{ mdx }}
+        >
+          <LivePreview />
+          <LiveEditor />
+          <LiveError />
+        </LiveProvider>
+      </div>
+    )
+  }
+
+  if (render) {
```

Comment:
@johno Apologies for pining you on this, but could you share where is this `render` props coming from? I'm trying to hunt down issues and I traced back to this original PR/commit, which introducing this `render` props. But I cannot see it anywhere in the docs. Could you point me somewhere? Thanks!